packages/integration-sdk-runtime/src/logger/index.ts

```diff
@@ -248,26 +259,24 @@ export class IntegrationLogger extends EventEmitter
   stepStart(step: StepMetadata) {
     const name = 'step_start';
     const description = `Starting step "${step.name}"...`;
-    this.info({ step: step.id }, description);
+    this.info({}, description);
```

Comment:
There is no need to pass in an empty object, it can be just `info(description)`.

Reply:
👍